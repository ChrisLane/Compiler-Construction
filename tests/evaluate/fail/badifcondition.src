// If conditions should result in a boolean. This should not evaluate.
main() {
    if (9001) {} else {};
}
