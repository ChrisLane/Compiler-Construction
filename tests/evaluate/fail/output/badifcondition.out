Fatal error: exception Failure("Invalid value for if condition.")
